apps/www/src/app/create/question-form/index.tsx

```diff
@@ -1,90 +1,101 @@
   question: Question;
   setQuestion: (question: Question) => void;
   remove: () => void;
 }
 
-interface PossibleAnswerControlProps {
+interface PossibleAnswerProps {
   possibleAnswer: string;
   isCorrectAnswer: boolean;
-  setPossibleAnswer: (
-    possibleAnswer: Omit<PossibleAnswerControlProps, 'setPossibleAnswer'>
-  ) => void;
+}
+
+interface PossibleAnswerControlProps extends PossibleAnswerProps {
+  setPossibleAnswer: (possibleAnswer: PossibleAnswerProps) => void;
 }
 
 const PossibleAnswerControl = ({
   possibleAnswer,
   isCorrectAnswer,
   setPossibleAnswer,
 }: PossibleAnswerControlProps) => {
+  const handleChange = ({ target: { value } }: ChangeEvent<HTMLInputElement>) =>
+    setPossibleAnswer({
+      possibleAnswer: value.toString(),
+      isCorrectAnswer,
+    });
+
+  const handleCheck = ({
+    target: { checked },
+  }: ChangeEvent<HTMLInputElement>) =>
+    setPossibleAnswer({
+      possibleAnswer,
+      isCorrectAnswer: !!checked,
+    });
+
   return (
     <div className="possible-answer">
       <TextField
         label="Answer"
-        className="answer"
         value={possibleAnswer}
-        onChange={(event) =>
-          setPossibleAnswer({
-            possibleAnswer: event.target.value,
-            isCorrectAnswer,
-          })
-        }
+        onChange={handleChange}
+        className="answer"
       />
       <Checkbox
+        checked={isCorrectAnswer}
         color="primary"
+        onChange={handleCheck}
         className="check"
-        checked={isCorrectAnswer}
-        onChange={(event) =>
-          setPossibleAnswer({
-            possibleAnswer,
-            isCorrectAnswer: event.target.checked,
-          })
-        }
       />
     </div>
   );
 };
 
 const QuestionForm = ({ question, setQuestion, remove }: QuestionFormProps) => {
+  const storage = getStorage();
+
   const handleSetPossibleAnswer = (
-    payload: Omit<PossibleAnswerControlProps, 'setPossibleAnswer'>,
-    index: number
-  ) => {
+    selectedAnswer: PossibleAnswerProps,
+    answerIndex: number
+  ) =>
     setQuestion({
       ...question,
       possibleAnswers: updatePrimitiveItemAt(
         question.possibleAnswers,
-        index,
-        payload.possibleAnswer
+        answerIndex,
+        selectedAnswer.possibleAnswer
       ),
-      correctAnswerIndex: payload.isCorrectAnswer
-        ? index
-        : question.correctAnswerIndex,
+      correctAnswerIndex: selectedAnswer.isCorrectAnswer ? answerIndex : question.correctAnswerIndex,
     });
-  };
 
-  const handleAttachmentChange = async (event: any) => {
-    console.log(event.target.files);
-    const files: FileList = event.target.files;
-    if (files.length) {
-      const file = files.item(0);
-      const storageRef = firebase.storage().ref(`${Date.now()}_${file.name}`);
-      await storageRef.put(file);
-      const url = await storageRef.getDownloadURL();
+  /** TODO
+   * Fix attaching file bug.
+   *  */    
```

Comment:
TODOs are easier to find like this
```suggestion
  // TODO: Fix attaching file bug.
```